interface/weno/formularyInsert.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * weno formulary insert
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+ require_once("../globals.php");
+ 
+ echo xlt("Trying")."<BR>";
+
+$zip = new ZipArchive;
+$zip->open('FormularyFiles.zip');
+$zip->extractTo('./');
```

Comment:
Permissions should not let you write to main codebase (which is a good thing).
Use the tmp folder instead; see the script for Administration-Backup for details.